docs/notebook_index.rst

```diff
@@ -10,5 +10,6 @@ directory.
 
     notebooks/Datasource Registry.ipynb
     notebooks/Development Examples.ipynb
+    notebooks/Diagnostic Walkthrough.ipynb
```

Comment:
@miclark your headers in your notebook become the outline in the docs, so right now the doc title shown in the index is "Checking for Installed rpms".  That may be what you want, but its different from the name of the file, so I'm just checking.

Reply:
Changed to "Diagnostic Walkthrough" to match the file name.